starter_code/src/movies.js

```diff
@@ -93,25 +93,28 @@ ratesAverage(movies);
 
 
 // Iteration 5: Drama movies - Get the average of Drama Movies
+//Can't solve more. Getting errors. Can't figure out.
+function dramaMoviesRate(arr) {
+  const sortDrama = arr.filter(function(e) {
+    return e.genre.includes("Drama");
+  });
 
-function getDramaMovies(arr) {
+	const dramaRatesArr = [];
 	
-	const sortDrama = arr.filter(function(e){
-		return e.genre.includes('Drama'); 
-		});
-		
-		if (rate => 8) {
-
-		};
+  for (let i = 0; i < sortDrama.length; i++) {
+    dramaRatesArr.push(sortDrama[i].rate);
+  }
 
+  const rateSum = dramaRatesArr.reduce((acc, num) => {
+    return acc + num;
+	}, 0);
 	
```

Comment:
here you can check, inside the reduce method, if "num" exists (if movie rate exists)

		if(!num) {
			return acc;
		} else {
			return acc + num;
		}